build/mixxx.py

```diff
@@ -39,8 +39,8 @@ def __init__(self, target, machine, build, toolchain, available_features=[]):
         if build is None:
             build = 'debug'
 
-        if not build in ['debug', 'release']:
-            raise Exception("invalid build type")
+        if not build in ['debug', 'release', 'none']:
```

Comment:
Noooooooooo please don't! A build is either release or debug.

What version of Qt do you link against in a none build? 
What macro do you define (MIXXX_BUILD_DEBUG or MIXXX_BUILD_RELEASE) -- that affects the runtime assertions. 
What MSVC runtime do you link against?

I think the right way to do this is via a separate scons option.